doc/lightningd-config.5

```diff
@@ -216,16 +216,16 @@ Note that once you encrypt the \fBhsm_secret\fR this option will be mandatory fo
 .SH Lightning node customization options
 
  \fBalias\fR=\fIRRGGBB\fR
```

Comment:
`alias` is not RRGGBB, maybe NAME suits here?

Reply:
Good catch